compiler/AST/FnSymbol.cpp

```diff
@@ -1025,6 +1025,10 @@ bool FnSymbol::isAnonymous() const {
   return hasFlag(FLAG_ANONYMOUS_FN) || hasFlag(FLAG_LEGACY_LAMBDA);
 }
 
+bool FnSymbol::isInline() const {
+  return hasFlag(FLAG_INLINE);
+}
```

Comment:
I do not think we need `isInline()`. Remove it from the PR (also in .h)?

Reply:
That's why I wrote `bool hasFlag(BaseAST* ast, int flag)` et al. :)

Reply:
Oh, perfect! 🎉  Yet another thing I don't know or forgot about! 😁 You re-learn things every day!